src/main.rs

```diff
@@ -22,7 +22,8 @@ fn validate_grading_system() {
 
 /// Calculate the grade
 fn calculate_grade(carrots: i32, nuts: i32, seeds: i32) -> i32 {
-    todo!()
+    let grade = (carrots + nuts) * (carrots + nuts + seeds);
+    return grade;
```

Comment:
In Rust, when returning from a function, we don't actually need to write the return keyword like `return x;` if it's the last expression. Instead, we can just do `x`. For example:

```rust
fn foo() -> i32 {
    let x = 5;
    x
}
```

Or, to simplify that more:

```rust
fn foo() -> i32 {
    5
}
```

Here are some more docs on that: https://doc.rust-lang.org/rust-by-example/fn.html